scripts/create_initial_tags.py

```diff
@@ -0,0 +1,250 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from flask_appbuilder import Model
+from sqlalchemy import create_engine
+from sqlalchemy.sql import func, functions, join, literal, select, union_all
+
+from superset import app
+from superset.models.tags import ObjectTypes, TagTypes
+
+
+def add_types(engine, metadata):
+    """
+    Tag every object according to its type:
+
+      INSERT INTO tagged_object (tag_id, object_id, object_type)
+      SELECT
+        tag.id AS tag_id,
+        slices.id AS object_id,
+        'chart' AS object_type
+      FROM slices
+      JOIN tag
+      ON tag.name = 'type:chart'
+      UNION ALL
+      SELECT
+        tag.id AS tag_id,
+        dashboards.id AS object_id,
+        'dashboard' AS object_type
+      FROM dashboards
+      JOIN tag
+      ON tag.name = 'type:dashboard'
+      UNION ALL
+      SELECT
+        tag.id AS tag_id,
+        saved_query.id AS object_id,
+        'query' AS object_type
+      FROM saved_query
+      JOIN tag
+      ON tag.name = 'type:query';
```

Comment:
Can this be run multiple times and reach the same final state? If that's not the case it would be great to modify so that it is the case. What if the feature flag goes on/off/on/off over time and we want to create the bulk of missing tags? It seems like adding a simple `LEFT JOIN (...) WHERE {remote table column} IS NULL` would enable that.

Reply:
If/when that's the case, we could add it as a cli command `sync-tags` or something maybe.

Reply:
Good point, currently this will fail if run multiple times. I'll fix it.